frontend/base/src/test/java/com/virtuslab/gitmachete/resourcebundles/junit/GitMacheteBundlePropertiesTestSuite.java

```diff
@@ -33,40 +39,33 @@ public void htmlProperties_should_have_correct_syntax() throws IOException {
   @Test
   public void properties_should_have_no_double_quote_wrapping() throws IOException {
 
-    Properties property = new Properties();
-    ClassLoader loader = Thread.currentThread().getContextClassLoader();
-    property.load(loader.getResourceAsStream(macheteBundleProperties));
+    Properties properties = loadProperties();
 
-    property.entrySet().stream()
-        .filter(prop -> ((String) prop.getValue()).contains("\""))
-        .forEach(t -> {
-          String key = (String) t.getKey();
-          String value = (String) t.getValue();
-          assertFalse(
-              "Key '${key}' has value wrapped in double quotes (\"). Remove unnecessary wrapping",
-              value.endsWith("\"") && value.startsWith("\""));
-        });
+    properties.forEach((keyObj, valueObj) -> {
+      String key = (String) keyObj;
+      String value = (String) valueObj;
+      assertFalse(
+          "Key '${key}' has value wrapped in double quotes (\"). Remove unnecessary wrapping",
+          value.endsWith("\"") && value.startsWith("\""));
+    });
   }
 
   @Test
   public void properties_should_contain_only_valid_single_quotes() throws IOException {
 
-    Properties property = new Properties();
-    ClassLoader loader = Thread.currentThread().getContextClassLoader();
-    property.load(loader.getResourceAsStream(macheteBundleProperties));
+    Properties properties = loadProperties();
 
-    property.entrySet().stream()
-        .forEach(t -> {
-          String key = (String) t.getKey();
-          String value = (String) t.getValue();
-          assertFalse(
-              "Key '${key}' has a format element ({number}), but contains single quote (')." +
-                  "Use double single quote instead ('')",
-              value.matches(".*\\{\\d+}.*") && value.matches(".*((^'[^'])|([^']'[^'])|([^']'$)).*"));
-          assertFalse(
-              "Key '${key}' has NO format element ({number}), but contains double single quote ('')." +
-                  "Use single quote instead (')",
-              !value.matches(".*\\{\\d+}.*") && value.matches(".*''.*"));
-        });
+    properties.forEach((keyObj, valueObj) -> {
+      String key = (String) keyObj;
+      String value = (String) valueObj;
+      assertFalse(
+          "Key '${key}' has a format element ({number}), but contains a single apostrophe (')." +
+              "Use a double apostrophe ('') instead",
+          value.matches(".*\\{\\d+}.*") && value.matches(".*(^'[^']|[^']'[^']|[^']'$).*"));
```

Comment:
Or maybe even clearer:
```suggestion
          value.matches(".*\\{\\d+}.*") && value.matches("^'[^'].*|.*[^']'[^'].*|.*[^']'$"));
```